pages/linux/rpmspec.md

```diff
@@ -0,0 +1,20 @@
+# rpmspec
+
+> Query a RPM spec file.
+> More information: <https://manned.org/rpmspec>.
+
+- List binary packages which would be generated from the rpm spec file:
```

Comment:
Should we use `specification`? :thinking: 
```suggestion
- List binary packages which would be generated from the rpm specification file:
```

Reply:
Its by default called the `spec` file .. and `.spec` is the extension.
I am pretty sure that would be short for `specification` but all the docs always mention `spec`